tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -288,6 +293,9 @@ def initial_mirror_config(self, mirror2, poolname, imagename, **kw):
             mode - mirror mode to be configured pool or image
             mirrormode - type of mirror configured journal or snapshot
         """
+        log.debug(
```

Comment:
is debug log required?

Reply:
IMHO, it helps in debugging and avoids initiating runs with pdb for most of the times.

Reply:
Resolving based on discussions in review meeting, please re-open if required